### ISSUE TITLE
Can't navigate dropdown with Screen Reader + space not working

### ISSUE THREAD

**Issue by Vanals** [bl00mber/react-phone-input-2#420]
Hi!

Referencing this issue https://github.com/bl00mber/react-phone-input-2/issues/250

I Still find it impossible to navigate through the dropdown with Safari + Voiceover through the following link:
https://bl00mber.github.io/react-phone-input-2.html

It read all the country in one go and the tabs button does not let you move you down.  Nor the country are being read by the SR.

It does not replicate the native select input behaviour. Not sure the fix is working?
<img width="947" alt="Screenshot 2021-05-25 at 17 43 02" src="https://user-images.githubusercontent.com/25549355/119536185-b3816f80-bd80-11eb-83e4-f1a086ccec20.png">

Also, the `space` button does not work on the select input. Is there any reason? It breaks the native behavior and aria-practice guidelines 1.2 https://www.w3.org/TR/wai-aria-practices-1.2/#button

@bl00mber  @sackeyjason

**Comment by diaslucia** [comment]
Hi there! Having the same issue here
